app/views/budgets/ballot/_progress_bar.html.erb

```diff
@@ -2,22 +2,32 @@
   <%= @budget.formatted_heading_price(@heading) %>
 </span>
 
-<div class="progress" role="progressbar" tabindex="0"
-     id="progress"
-     aria-valuenow="0" aria-valuemin="0" aria-valuemax="100">
-  <div class="progress-meter"
-       style="width:
-       <%= progress_bar_width(@budget.heading_price(@heading),
-                              @ballot.amount_spent(@heading)) %>">
+<% if @heading.group_approval_voting? %>
+  <div class="panel">
+    <p style="padding:2rem">
```

Comment:
It's better don't use inline styles. Please change to `<p class="padding-top">` 🙏 😌

Reply:
Will do.

Reply:
@decabeza There doesn't seem to be a style definition for `padding-top`. Then it occured to me that the spacing the progress bar has to be from clearing the float (amount to the right). 

If we add a `.clear` class to the panel (`<div class="panel clear"></div>`) element it will look some thing like [this](https://www.dropbox.com/s/n2oudcv1g1kmsrk/Screenshot_2018-10-20%20Participatory%20budgeting.png?dl=0).

Is that alright?